starknet_py/tests/e2e/fixtures/account_clients.py

```diff
@@ -292,6 +294,70 @@ def new_account_client(request) -> AccountClient:
     return request.getfixturevalue(request.param)
 
 
+@pytest.fixture(scope="module")
+def gateway_account(
+    new_address_and_private_key: Tuple[str, str], gateway_client: GatewayClient
+) -> BaseAccount:
+    """
+    Returns a new Account created with GatewayClient
```

Comment:
Are we commited to putting dots after every sentence in docstrings?

Reply:
We should be, but I forget at times. Will fix that.